utils/download_sqlite_all.js

```diff
@@ -50,9 +51,19 @@ function download(callback) {
   const generateCommand = (sqlite, directory) => {
     let extract;
     if (/\.db\.bz2$/.test(sqlite.name_compressed)) {
-      extract = `bunzip2`;
-    } else if(/\.db\.tar\.bz2$/.test(sqlite.name_compressed)) {
-      extract = `tar -xjO`;
+      // Check if we have lbzip2 installed
+      if (commandExistsSync('lbzip2')) {
+        extract = 'lbzip';
+      } else {
+        extract = 'bunzip';
```

Comment:
according tar man page, flag `-j` is for `bzip2` not `bzip`. I belive that this is just typo and here we should has `bunzip2` command as it was on my original PR

Reply:
Makes sense. I just put in #486 to fix line 56, which had a similar error. Want to open another PR? ;)

Reply:
Sure, updated in #484